app/komponenter/veilederhilsen/veilederhilsen.tsx

```diff
@@ -1,20 +1,23 @@
 import { GuidePanel, Heading } from '@navikt/ds-react';
-
 import css from './veilederhilsen.module.css';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from '../tekstBlokk/tekstBlokk';
+
+interface VeilederHilsenProp {
+  innhold: SanityDokument | undefined;
```

Comment:
Nå kunne vi egentlig henta dataene componenten trenger i komponenten, så slipper vi å sende dem ned

Reply:
Ble enig med Sara om at vi for nå kan sende når det bare er ett steg, om det er greit?